src/middlewared/middlewared/plugins/pool_/import_disk.py

```diff
@@ -51,67 +118,65 @@ async def import_disk(self, job, device, fs_type, fs_options, dst_path):
         try:
             os.makedirs(src)
```

Comment:
I'd really rather make all of this synchronous. Forcing all of this to be asynchronous doesn't help us in any way and only complicates everything unnecessarily. All of this is file/disk I/O which is, inherently, non-friendly to asyncio.

Also, there are still tons of areas in these methods that are calling blocking I/O in the main event loop. This is also why I'd rather make this all synchronous.